tables/TaQL/ExprNodeSet.h

```diff
@@ -365,7 +370,7 @@ class TableExprNodeSet : public TableExprNodeRep
     Bool isBounded() const;
 
     // Get the number of elements.
-    uInt nelements() const;
+    uInt size() const;
```

Comment:
Can we keep nelements() as an alias? That would of simplify integrating these changes with CASA.